### DISCUSSION TITLE
Noise filter

### DISCUSSION THREAD

**Discussion by satopian** [bitbof/klecks#34] [Q&A] [answered]
I understand that kleki and Klecks are not exactly the same.
A noise filter has been added to kleki, won't it be implemented in Klecks?
The noise function is a great feature that can be used for various purposes such as textures, layer effects, etc.
Thank you.

**Comment by bitbof** [comment, marked as answer]
Hey satopian, noise is coming soon (this week) to Klecks, I just wanted to refactor the code a bit before making it public. Greets

**Reply by satopian** [reply]
That's excellent. Thank you.
